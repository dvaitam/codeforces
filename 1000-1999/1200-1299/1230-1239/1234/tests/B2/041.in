22 5
4 7 9 3 19 17 3 3 7 6 17 14 1 19 12 16 10 8 7 20 16 8
