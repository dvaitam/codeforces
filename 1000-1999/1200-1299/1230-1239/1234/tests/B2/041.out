5
20 7 8 10 16
